libcxx/test/std/containers/associative/map/map.access/at.pass.cpp

```diff
@@ -42,11 +43,16 @@ int main(int, char**) {
     assert(m.at(4) == 4.5);
     assert(m.at(5) == 5.5);
 #ifndef TEST_HAS_NO_EXCEPTIONS
+
+// throwing is not allowed in constexpr
+#  if TEST_STD_VER < 26
```

Comment:
This should still be tested during runtime. You can use something like
```
#if TEST_STD_VER >= 26
  if (!std::is_constant_evaluated())
#endif
```
Throughout.

Reply:
Fixed as of ad96d233ce46